src/test/java/com/znsio/rpap/AppBaseTest.java

```diff
@@ -0,0 +1,58 @@
+package com.znsio.rpap;
```

Comment:
Done

Reply:
In suiteSetup -calling a single **getDriver** method which will launch webDriver in case of web and AppiumDriver in case of Apps

In afterSuite - calling **killDriver** method - which will quit the webdriver in case of web and stop Appium driver in case of Apps